quarkchain/evm/messages.py

```diff
@@ -187,6 +188,16 @@ def validate_transaction(state, tx):
     return True
 
 
+def _refund(state, message, total, refund_percent):
+    to_refund = total * refund_percent // 100
+    to_burn = total - to_refund
+    state.delta_token_balance(message.sender, message.gas_token_id, to_refund)
+
+    # burn
+    if to_burn:
+        state.delta_token_balance(bytes(20), message.gas_token_id, to_burn)
```

Comment:
I would suggest using another constant address rather than bytes(20) to avoid confusion with special contract addresses.